### ISSUE TITLE
E713 not correct detected

### ISSUE THREAD

**Issue by spaceone** [hhatto/autopep8#262]
[e713.py](https://github.com/hhatto/autopep8/files/555950/e713.txt)

```
$ ~/pycodestyle.py --select E713 /tmp/e713.py
/tmp/e713.py:4:6: E713 test for membership should be 'not in'
$ ~/autopep8.py -aaa --select E713 /tmp/e713.py
class Foo(object):
        def validate_name(self):
                super(VLAN, self).validate_name()
                if not '.' in self.name:
                        raise DeviceError(_('Invalid device name: %r') % (self.name,))
                if not (1 <= self.vlan_id <= 4095):
                        raise DeviceError(_('Invalid VLAN ID. Must be between 1 and 4095.'), self.name)
```

**Comment by spaceone** [comment]
Another broken example:
[e713-2.py](https://github.com/hhatto/autopep8/files/555955/e713.txt)

**Comment by spaceone** [comment]
And another one:
"\t\t\tif not isinstance(self.interfaces[name], Ethernet) or not name in PHYSICAL_INTERFACES:"

**Comment by spaceone** [comment]
After your last fixes there is still this one:
```
$ printf "if not 'DC IP' in info:\n" | ~/autopep8.py  -aaa --select E713,E714 - | ~/pycodestyle.py --select E713,E714 -
stdin:1:4: E713 test for membership should be 'not in'
```

**Comment by hhatto** [comment]
fix in efe4aa3c4258281adae0a058cae79917a2634cba
